cli/src/serve.js

```diff
@@ -229,6 +229,11 @@ const initialize_servers = (ctor, opts) => {
           console.info(`Static files being served from ${opts.serve_static}`);
         }
         srv.on('request', fileServer(opts.serve_static));
+      } else {
```

Comment:
The server removes all existing request listeners and puts itself in their place, then passes down anything that it doesn't handle:

```
const extant_listeners = server.listeners('request').slice(0);
      server.removeAllListeners('request');
      server.on('request', (req, res) => {
        const req_path = url.parse(req.url).pathname;
        if (req_path.indexOf(`${this._path}/`) === 0) {
          const sub_path = req_path.replace(path_replace, '');
          const handler = this._http_handlers.get(sub_path);
          if (handler !== undefined) {
            logger.debug(`Handling HTTP request to horizon subpath: ${sub_path}`);
            return handler(req, res);
          }
        }
        if (extant_listeners.length === 0) {
          res.statusCode = 404;
          res.write('File not found.');
          res.end();
        } else {
          extant_listeners.forEach((l) => l.call(server, req, res));
        }
      });
```